lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -65,32 +65,35 @@ class ComicFormViewModel {
     // MARK: - Variables
 
     var comicFormMode: ComicFormMode
-
+    var coverPhoto: UIImage?
     var navigationBarTitle: String {
         return comicFormMode == .add ? "Add Book" : "Edit Book"
     }
+
     var genreTitles: Observable<[String]> {
         return Observable.just(Comic.Genre.allCases.map { $0.title })
     }
+
     var conditionTitles: Observable<[String]> {
         return Observable.just(Comic.Condition.allCases.map { $0.title })
     }
 
-    var coverPhoto: UIImage?
-
     // MARK: - Private Variables
 
     private let disposeBag = DisposeBag()
     private let lucienAPIClient = LucienAPIClient()
+    private let comicID: String?
 
     // ComicFormMode is set to .add by default if default init is used.
     init() {
         comicFormMode = .add
+        comicID = ""
     }
 
     /// Initializes ComicFormViewModel with a ComicFormMode of edit.
     /// The parameters will be used by the ComicFormViewController to fill in its textfields.
-    init(coverPhoto: UIImage?,
+    init(comicID: String,
```

Comment:
Is this accessible from anywhere in the app yet? We should probably create an initializer that takes in a `Comic` rather than this one, because if you think about it, when you click on a comic to edit, you'll have the whole object, not just the individual fields.